### PR TITLE
Seek gesture

### PR DESCRIPTION
**Pull request by gwjgwj**
Added a gesture in TrackActivity, that allows moving forward and backwards in the current track. It is useful for precise seek on long tracks.

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/musicplayer/activities/TrackActivity.kt [hunk 1]
```diff
@@ -286,6 +289,15 @@ class TrackActivity : SimpleActivity(), PlaybackSpeedListener {
                     finish()
                     activity_track_top_shadow.animate().alpha(0f).start()
                     overridePendingTransition(0, R.anim.slide_down)
+                } else
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/musicplayer/activities/TrackActivity.kt:292]: oh that is too bad code, stick to the code style used elsewhere please

Comment (gwjgwj) [reply app/src/main/kotlin/com/simplemobiletools/musicplayer/activities/TrackActivity.kt:292]: I have reformatted and modified code a little. If this is not what you meant, treat it as a feature request.

**Comment by tibbi** [comment]
ah this feels really strange, and really not needed on a screen with a seekbar...

**Comment by gwjgwj** [comment]
For recordings ca. 2 hours long the seekbar has too little precision to be useful. Similar approach is used in VLC for video playback.

**Comment by tibbi** [comment]
so just click on current or max duration to fast forward 15 seconds

**Pull request closed without merging**